src/client/api/pythonApi.ts

```diff
@@ -316,6 +317,19 @@ export class PythonInstaller implements IPythonInstaller {
             });
         }
     }
+
+    public async isProductVersionCompatible(
+        product: Product,
+        semVerRequirement: string,
+        resource?: InterpreterUri
+    ): Promise<ProductInstallStatus> {
+        try {
+            const api = await this.apiProvider.getApi();
+            return await api.isProductVersionCompatible(product, semVerRequirement, resource);
```

Comment:
You don't need the await
```suggestion
            return api.isProductVersionCompatible(product, semVerRequirement, resource);
```